src/ServiceProvider.php

```diff
@@ -5,12 +5,11 @@
 
 class ServiceProvider extends SocialiteServiceProvider
 {
-
     /**
      * {@inheritdoc}
      */
     public function boot()
     {
-        \Event::fire(new SocialiteWasCalled());
+        event(new SocialiteWasCalled());
```

Comment:
We could also use method injection.  That would probably be the cleanest OO wise.

[Method injection works off of the ServiceProvider->boot()](https://mattstauffer.co/blog/laravel-5.0-method-injection#when-else-will-the-container-resolve-a-method)

For example:

```php

public function boot(Illuminate\Contracts\Events\Dispatcher $event)
{
    $event->fire(new SocialiteWasCalled());  
}
```